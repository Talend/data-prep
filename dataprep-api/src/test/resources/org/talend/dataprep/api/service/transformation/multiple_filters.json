{
  "actions": [
    {
      "action": "uppercase",
      "parameters": {
        "column_id": "0002",
        "column_name": "lastname",
        "scope": "column",
        "filter": {
          "eq": {
            "field": "0005",
            "value": "true"
          }
        }
      }
    },
    {
      "action": "uppercase",
      "parameters": {
        "column_id": "0001",
        "column_name": "lastname",
        "scope": "column",
        "filter": {
          "eq": {
            "field": "0005",
            "value": "false"
          }
        }
      }
    }

  ]
}