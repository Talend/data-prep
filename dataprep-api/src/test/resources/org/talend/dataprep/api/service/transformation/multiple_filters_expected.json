{
  "records": [
    {
      "0000": "1",
      "0001": "JOHN",
      "0002": "Lennon",
      "0003": "40",
      "0004": "10/09/1940",
      "0005": "false",
      "tdpId": 1
    },
    {
      "0000": "2",
      "0001": "David",
      "0002": "BOWIE",
      "0003": "67",
      "0004": "01/08/1947",
      "0005": "true",
      "tdpId": 2
    },
    {
      "0000": "3",
      "0001": "Dylan",
      "0002": "Bob",
      "0003": "74",
      "0004": "empty",
      "0005": "",
      "tdpId": 3
    },
    {
      "0000": "4",
      "0001": "HENDRIX",
      "0002": "Jimi",
      "0003": "",
      "0004": "1942-27-11",
      "0005": "false",
      "tdpId": 4
    }
  ]
}